src/CloudFlare/Zone/Settings.php

```diff
@@ -471,6 +471,8 @@ public function change_ipv6($zone_identifier, $value = 'off') {
 	 * @return array|mixed
 	 */
 	public function change_minify($zone_identifier, $value) {
+		
+		$data = ['value' => $value];
```

Comment:
Ahhh I've updating methods to be more like this: 
\CloudFlare\Zone::zones:
```php

	/**
	 * List zones permission needed: #zone:read
	 * List, search, sort, and filter your zones
	 *
	 * @param string  $name      A domain name
	 * @param string  $status    Status of the zone (active, pending, initializing, moved, deleted)
	 * @param integer $page      Page number of paginated results
	 * @param integer $per_page  Number of zones per page
	 * @param string  $order     Field to order zones by (name, status, email)
	 * @param string  $direction Direction to order zones (asc, desc)
	 * @param string  $match     Whether to match all search requirements or at least one (any) (any, all)
	 *
	 * @return array|mixed
	 */
	public function zones($name = '', $status = 'active', $page = 1, $per_page = 20, $order = 'status', $direction = 'desc', $match = 'all') {

		$data = [
			'name' => $name,
			'page' => $page,
			'per_page' => $per_page,
			'order' => $order,
			'direction' => $direction,
			'match' => $match
		];

		if (!empty($status)) {
			$data['status'] = $status;
		}

		return $this->get('zones', $data);

	}
```

I'll make an effort to go through and update the methods so they all have all the supported / default options.  Don't you think that would be better?!?